### PR TITLE
Fixed pointing feature in mc

### PR DESCRIPTION
**Pull request by snyxan**
# Description

Pointing is partially broken in mc. This PR fixed it.

Fixes # (issue)

#95 #84 

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [ ] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change

## Type of requested review

- [ ] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

Before: only one block of MC BlockObject will flash when pointed

After: the whole BlockObject will flash

# Testing

Tested locally.

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [x] I have added relevant collaborators to review the PR before merge.

### CODE REVIEW

**Review by aszlam — CHANGES_REQUESTED** [review]
shouldn't we handle the relative locations (lines 13-16)?  what about "point to the left of the cow" ?

**Comment by snyxan** [comment]
> shouldn't we handle the relative locations (lines 13-16)? what about "point to the left of the cow" ?

You are right. I misunderstood it at first.

Pushed a simple fix for it. However, I cannot test it locally since parser failed at parsing it. Created an issue for that.

**Comment by anuragprat1k** [comment]
> > shouldn't we handle the relative locations (lines 13-16)? what about "point to the left of the cow" ?
> 
> You are right. I misunderstood it at first.
> 
> Pushed a simple fix for it. However, I cannot test it locally since parser failed at parsing it. Created an issue for that (#112).

You might have already tried this - but I get around nsp failures like this by manually adding the command and its GT parse to the GT text file in the datasets folder. That's a pretty quick way to unblock testing.

**Comment by kavyasrinet** [comment]
Thanks for pointing that out @anuragprat1k . We should also try the adoption of the autocomplete tool Rebecca built - the goal of that tool is to unblock us in scenarios like these.

**Comment by snyxan** [comment]
@aszlam does that look good now?

File: craftassist/agent/dialogue_objects/point_target.py
Comment (anuragprat1k) [inline craftassist/agent/dialogue_objects/point_target.py:24]: @snyxan curious, is there ever a scenario in which `len(pointed_target)` is 3? My understanding from when I was doing this change for the locobot was that it is never the case in minecraft.

Comment (snyxan) [reply craftassist/agent/dialogue_objects/point_target.py:24]: as discussed in Wednesday's coding block: any loc from spatial reasoning sub-interpreter will give (x, y, z) and and loc directly from memory node's get_point_at_target() fn will give (x1, x2, y1, y2, z1, z2).

**Comment by kavyasrinet** [comment]
What's going on with this PR @snyxan ? Are we planning on merging/updating this soon ?

**Comment by snyxan** [comment]
> What's going on with this PR @snyxan ? Are we planning on merging/updating this soon ?

waiting for an approval here. Tag Arthur to review it @aszlam.

**Review by anuragprat1k — APPROVED** [review]
looks good to me.

**Comment by snyxan** [comment]
close this one, see #656

**Pull request closed without merging**
